utilities/load-test/loadtest/utility.go

```diff
@@ -61,8 +70,106 @@ type E2EFixtureK8sClient struct {
 	AppClientset     appclientset.Interface
 }
 
-// func CheckError(err error) {
-// 	if err != nil {
-// 		debug.PrintStack()
-// 	}
-// }
+// Kubectl_apply is basically a go utility that performs kubetcl apply -n namespace -f  *yaml
+func Kubectl_apply(namespace string, URL string) {
+	prg := "kubectl apply -n %s -f %s"
+	// namespace := "argocd"
+	// URL := "https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml"
+	s := fmt.Sprintf(prg, namespace, URL)
+
+	// To print which command is running
+	fmt.Println("Running: ", s)
+
+	// To get the output of the command
+	out, err := exec.Command("kubectl", "apply", "-n", namespace, "-f", URL).Output()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// To actually run the command (runs in background)
+	cmd_run := exec.Command("kubectl", "apply", "-n", namespace, "-f", URL)
+	err_run := cmd_run.Run()
+
+	if err_run != nil {
+		log.Fatal(err_run)
+	}
+	fmt.Println(string(out), "Command Run Successful!")
+}
+
+// The PodRestartcount is used to return the count of the pods (Basically, RESTART count from kubectl get pods)
+func PodRestartcount(namespace string) map[string]string {
+	var allrestartInfo = make(map[string]string)
+
+	out, err := exec.Command("kubectl", "get", "pods", "-n", namespace).Output()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	res := string(out)
+
+	for index, i := range strings.Split(res, "\n") {
+		if index != 0 && index < len(strings.Split(res, "\n"))-1 {
+			allrestartInfo[strings.Fields(i)[0]] = strings.Fields(i)[3]
+		}
+	}
+
+	return allrestartInfo
+}
+
+// The Get_pod_info tells us the memory usage of the pod along with container name
+func Get_pod_info(kubeconfig string, master string, namespace string, podName string) map[string]interface{} {
+	podResource := make(map[string]interface{})
+	config, err := clientcmd.BuildConfigFromFlags(master, kubeconfig)
+	if err != nil {
+		panic(err)
+	}
+
+	mc, err := metrics.NewForConfig(config)
+	if err != nil {
+		panic(err)
+	}
+
+	if podName != "" {
+		// To get memory info of the specific pod passed as an argument
+		podMetrics, err := mc.MetricsV1beta1().PodMetricses(namespace).Get(context.TODO(), podName, metav1.GetOptions{})
+		if err != nil {
+			panic(err)
+		}
+		podResource[(podMetrics.ObjectMeta.Name + "\t" + podMetrics.Containers[0].Name)] = podMetrics.Containers[0].Usage["memory"].ToUnstructured()
```

Comment:
It looks like you are calling `ToUnstructed()` here;  if you instead _don't_ call `toUnstructured`, it means you shouldn't have to parse it as a string later.